### ISSUE TITLE
Google Gemini exploration

### ISSUE THREAD

**Issue by jakedetels** [lgrammel/modelfusion#244]
Contribution suggestion from @lgrammel 

"Google Gemini exploration (this might be impact API design, so I want to prob do it myself)"

**Comment by Morphexe** [comment]
Am I correct to think that there is no support for gemini models/vertex on the lib at this moment? If one wanted to add this, what would be the steps to go?

**Comment by lgrammel** [comment]
@Morphexe correct. if you want to add this, you can follow the implementation pattern for any of the existing text generators and create a version of the gemini api. if you want to use vertex, you probably need to use google auth library, which is not edge ready, so the best approach would be to create a separate node module.

**Comment by aleclarson** [comment]
Google has OpenAI-compatible endpoint

https://ai.google.dev/gemini-api/docs/openai
